src/cbor.h

```diff
@@ -45,7 +45,17 @@
 #ifdef __cplusplus
 extern "C" {
 #else
-#include <stdbool.h>
+#if defined __STDC_VERSION__ && __STDC_VERSION__ >= 199901L
+#  include <stdbool.h>
+#else
+   typedef int bool;
+#  ifndef true
+#    define true 1
+#  endif
+#  ifndef false
+#    define false 0
+#  endif
```

Comment:
This is correct, but such not a good idea in a public header.

I think we're just going ban C89 builds.